proposals/improved-interpolated-strings.md

```diff
@@ -138,14 +155,21 @@ to be converted to an _applicable\_interpolated\_string\_builder\_type_. There a
 1. A method argument is converted as part of determining applicable function members (covered below), or
 2. Given an _interpolated\_string\_expression_ `S` being converted to type `T`, the following is true:
     * `T` is an _applicable\_interpolated\_string\_builder\_type_, and
-    * `T` has an accessible static bool-returning method `GetInterpolatedStringBuilder` that takes 2 int parameters and 1 out parameter of type `T`, in that order.
+    * One of the following is true:
+        * `T` has an accessible static T-returning method `Create` that takes 2 int parameters and 1 out parameter of type `bool`, in that order.
+        * `T` has an accessible static T-returning method `Create` that takes 2 int parameters, in that order.
```

Comment:
What about the other state that's passed in?  I know you have attributed arguments in open issues (even though I think it's really important), but your examples allow passing in the receiver, and that doesn't seem to be factored in here...?

Reply:
Neither of these is applicable there. Those are bullet one, covered in applicable function member.

Reply:
I don't follow, but you think it's fine, so ok.

Reply:
Basically: these bullets don't allow for state to come in. The version that allows state to come in is covered in the applicable function member section.